scripts/benchmark_ipca.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
I wonder if this file belongs in the examples folder instead?

Reply:
Removed for now.